ui/components/multichain/select-action-modal-item/select-action-modal-item.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  AlignItems,
+  BackgroundColor,
+  Display,
+  FlexDirection,
+  IconColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import {
+  AvatarIcon,
+  AvatarIconSize,
+  Box,
+  Icon,
+  IconName,
+  IconSize,
+  Text,
+} from '../../component-library';
+import Tooltip from '../../ui/tooltip';
+
+export const SelectActionModalItem = ({
+  actionIcon,
+  onClick,
+  showIcon,
+  primaryText,
+  secondaryText,
+  isDisabled,
+  tooltipTitle,
+}) => {
+  const SelectActionItem = () => {
+    return (
+      <Box
+        paddingTop={4}
+        paddingBottom={4}
+        gap={4}
+        display={Display.Flex}
+        flexDirection={FlexDirection.Row}
+        as="a"
+        href="#"
+        onClick={(e) => {
+          e.preventDefault();
+          onClick();
+        }}
+        className={`select-action-modal-item ${
+          isDisabled && 'select-action-modal-item__disabled'
+        }`}
+        data-testid="select-action-modal-item"
+      >
+        <Box>
+          <AvatarIcon
+            iconName={actionIcon}
+            color={IconColor.primaryInverse}
+            backgroundColor={BackgroundColor.primaryDefault}
+            size={AvatarIconSize.Md}
+            className="select-action-modal-item__avatar"
+          />
+        </Box>
+        <Box display={Display.Flex} flexDirection={FlexDirection.Column}>
+          <Box
+            display={Display.Flex}
+            flexDirection={FlexDirection.Row}
+            gap={2}
+            alignItems={AlignItems.center}
+          >
+            <Text variant={TextVariant.bodyLgMedium}>{primaryText}</Text>
+            {showIcon && (
+              <Icon
+                name={IconName.Export}
+                size={IconSize.Xs}
+                color={IconColor.iconAlternative}
+              />
+            )}
+          </Box>
+          <Text variant={TextVariant.bodyMd}>{secondaryText}</Text>
+        </Box>
+      </Box>
+    );
+  };
+  return (
+    <>
+      {isDisabled ? (
+        <Tooltip position="bottom" title={tooltipTitle}>
+          <SelectActionItem />
+        </Tooltip>
+      ) : (
+        <SelectActionItem />
+      )}
+    </>
+  );
+};
+
+SelectActionModalItem.propTypes = {
+  /**
+   * Show link icon with text
+   */
+  showIcon: PropTypes.bool,
+  /**
+   * onClick handler for each action
+   */
+  onClick: PropTypes.func,
```

Comment:
I think this should be required, they should all have actions

Reply:
We should require most of these, now that I think about it.

Reply:
Yup! Updated here https://github.com/MetaMask/metamask-extension/pull/20559/commits/8d602620ce1bdf9f6b57d9e53f40aa8f8cb9580e